pkgs/top-level/aliases.nix

```diff
@@ -217,6 +217,7 @@ mapAliases ({
   keymon = throw "keymon has been removed from nixpkgs, as it's abandoned and archived."; # 2019-12-10
   kvm = qemu_kvm; # added 2018-04-25
   latinmodern-math = lmmath;
+  ldc2 = ldc; # added 2020-04-18
```

Comment:
just curious why this alias is needed?

Reply:
I had added the alias because `ldc2` is the name used by the project for the packages on its releases page https://github.com/ldc-developers/ldc/releases

Reply:
Can you remove it?
I ask because, they may name their packages like this, but it looks superfluous to do this as well when no other distro (in repology at least) calls it ldc2 https://repology.org/project/ldc/versions. (ldc2 doesn't even show up).

Reply:
Done in 71fe24e